app/src/main/java/org/mozilla/fenix/components/FenixSnackbar.kt

```diff
@@ -0,0 +1,98 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.components
+
+import android.util.TypedValue
+import android.view.View
+import android.view.ViewGroup
+import com.google.android.material.snackbar.BaseTransientBottomBar
+import android.view.LayoutInflater
+import androidx.core.widget.TextViewCompat
+import kotlinx.android.synthetic.main.fenix_snackbar.view.*
+import org.mozilla.fenix.R
+import org.mozilla.fenix.ext.increaseTapArea
+
+class FenixSnackbar private constructor(
+    parent: ViewGroup,
+    content: View,
+    contentViewCallback: FenixSnackbarCallback
+) : BaseTransientBottomBar<FenixSnackbar>(parent, content, contentViewCallback) {
+
+    init {
+        view.background = null
+        content.snackbar_btn.increaseTapArea(increaseTouchableAreaBy)
```

Comment:
nit: the rest of our `increaseTapAreas` pass params with names like: `closeButtonIncreaseDps`. Perhaps we should mirror this for consistency?